packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -92,9 +92,10 @@ import {
     IEntityTypeOption,
     IParentOption,
 } from './models';
+import { getAltUnitKeys } from '../../util/measurement';
 
 const ENTITY_GRID_ID = 'entity-insert-grid-data';
-const ALIQUOT_FIELD_COLS = ['aliquotedfrom', 'name', 'description', 'samplestate'];
+const ALIQUOT_FIELD_COLS = ['aliquotedfrom', 'name', 'description', 'samplestate', 'storedamount', 'units', 'foo'];
```

Comment:
what is 'foo'?

Reply:
Since we are adding storedamount and units, we probably might as well add materialexpdate here.

Reply:
Oops. "foo" was accidental. materialexpdata added.